site/content/docs/5.0/forms/file.md

```diff
@@ -50,24 +48,114 @@ Longer placeholder text is truncated and an ellipsis is added when there's not e
 
 We hide the default file `<input>` via `opacity` and instead style the `<label>`, and declare a `width` and `height` on the `<input>` for proper spacing for surrounding content.
 
+### Multiple
+
+To enable multiple file selection just add `multiple` on your `<input>`
+
+{{< example >}}
+<div class="form-file" data-toggle="file-input">
+  <input type="file" class="form-file-input" id="customFileMultiple" multiple>
+  <label class="form-file-label" for="customFileMultiple">
+    <span class="form-file-text">Choose files...</span>
+    <span class="form-file-button">Browse</span>
+  </label>
+</div>
+{{< /example >}}
+
+### Handle form reset
+
+File inputs will be reset on a form's `reset` event.
+
+{{< example >}}
+<form id="formFileInput">
+  <div class="form-file" data-toggle="file-input">
+    <input type="file" class="form-file-input" id="customFileForm">
+    <label class="form-file-label" for="customFileForm">
+      <span class="form-file-text">Choose file...</span>
+      <span class="form-file-button">Browse</span>
+    </label>
+  </div>
+</form>
+<button id="btnResetFormFileInput" class="btn btn-primary mt-3">
+  Reset form
+</button>
```

Comment:
Could this be moved inside the form with `type="reset"`?

Reply:
yep the place doesn't really matter

Reply:
This is still pending AFAICT @Johann-S :)

Reply:
I wasn't sure if @mdo would like to tackle that or not